src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -1255,7 +1314,8 @@ public void testDataTypes() throws SQLException, ParseException {
       throw new RuntimeException("Cannot create table " + tableName, e);
     }
 
-    SnowflakeStreamingIngestChannel channel = openChannel(tableName, "CHANNEL");
+    SnowflakeStreamingIngestChannel channel =
```

Comment:
Are we losing coverage for ON_ERROR=CONTINUE now?

Reply:
This test wasn't using `CONTINUE`, only ABORT. For the newly added test, I needed `CONTINUE`, so I added `OnErrorOption` as `openChannel` method parameter.